### PR TITLE
update French Polynesia phone format (8 digits)

### PR DESCRIPTION
**Pull request by ypelud**
add 2 digits to French Polynesia: 8 digits since sept-2014 (http://en.wikipedia.org/wiki/Telephone_numbers_in_French_Polynesia)

### CODE REVIEW

File: spec/functional/plausibility_spec.rb [hunk 1]
```diff
@@ -579,7 +579,7 @@ def self.it_is_correct_for(country_name, options={})
       it_is_correct_for 'Faroe Islands', :samples => '+298  969 597'
       it_is_correct_for 'Fiji (Republic of)', :samples => '+679  998 2441'
       it_is_correct_for 'French Guiana (French Department of)', :samples => '+594 594 123 456'
-      it_is_correct_for "French Polynesia (Territoire français d'outre-mer)", :samples => '+689  872 784'
+      it_is_correct_for "French Polynesia (Territoire français d'outre-mer)", :samples => '+689  872 784 00'
```
Comment (floere) [inline spec/functional/plausibility_spec.rb]: Should this not be in a 2-2-2-2 grouping? (I mean, it passes, but for not confusing readers)

**Comment by floere** [comment]
Great! I had one comment, before happily merging :)

File: spec/functional/plausibility_spec.rb (hunk 1 above)
Comment (ypelud) [inline spec/functional/plausibility_spec.rb]: yes of course, do you want another pull request or you make the change ?

**Comment by floere** [comment]
Just push another commit to your branch :)

File: spec/functional/plausibility_spec.rb (hunk 1 above)
Comment (floere) [inline spec/functional/plausibility_spec.rb]: I'd be happy if you just added a commit with the change to your fork's master branch. Then that commit will automatically appear on this PR.

**Comment by ypelud** [comment]
ok done

**Comment by floere** [comment]
Released in 2.10.3.
